src/styles/pages/_carbon-metrics.scss

```diff
@@ -0,0 +1,84 @@
+.docs-table-header-sortable {
+  .bsds-button-subtle {
+    padding: 0;
+    color: $text-color-primary;
+
+    &:hover {
+      background: transparent;
+      color: $text-color-primary;
+      text-decoration: underline;
+    }
+  }
+}
+
+.docs-carbon-filters {
+  display: flex;
+
+  dd {
+    margin-left: 1ch;
+  }
+}
+
+.docs-metrics-tile-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: space(6x);
+  margin-top: space(4x);
+}
+
+.docs-metrics-tile {
+  display: flex;
+  flex-direction: column;
+  border: $border-width-xxs solid $border-color-subtle;
+  padding: space(3x);
+  text-align: center;
+
+  .docs-metrics-trend-up {
+    fill: $feedback-success-40;
+  }
+
+  .docs-metrics-trend-down {
+    fill: $feedback-error-40;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+}
+
+.docs-metrics-tile-description {
+  @include font-size($fs-body-subtle);
+  margin-bottom: auto;
```

Comment:
is there a bottom margin coming from something?

Reply:
This pushes the flex items below it down so that things line up nice when text wraps.

Reply:
thoughts on adding a code comment explaining this?